### ISSUE TITLE
Sending a function name to the chat spills JS code into chat

### ISSUE THREAD

**Issue by phylll** [michael-buschbeck/mychs-macro-magic#166]
`!mmm chat: ${chat}` spills JS code into the chat:

```
phyllo:function functionImplWithVariables(...args)
{
return value.apply(variables, args);
}
```

`!mmm   debug chat: ${playerMessage} ${chat}` makes it look more confusing:

**(From Mych's Macro Magic):**
🔎 "/w \"Yerrick MacRothgar\" Ahhhh -- da ist dir gerade ein kalter Schauer den Rücken hinuntergelaufen!" function functionImplWithVariables(...args)
**Mych's Macro Magic:** {
return value.apply(variables, args);
}
**(From phyllo):** Ahhhh -- da ist dir gerade ein kalter Schauer den Rücken hinuntergelaufen!

I have a vague idea now what I was trying to do there, and I think at the time ${chat} just told me the function name, if it was overwritten with `whisperback()` or not. I'm not doing that any more, anyway, and the current version of the `senseLogic` script does not reproduce the bug.

**Comment by michael-buschbeck** [comment]
Ah, overwriting `chat` with ` whisperback` doesn't work anymore anyway (because `chat()` has different parameters since the recent "impersonate sender" change).

Is/was this a useful feature for you? I could try to revive it then (by matching up `whisperback()`s parameters again).

**Comment by phylll** [comment]
No, please don't. I never seriously used it, just played around with it once and apparently forgot to remove debugging code from the version that ended up in the live game 👎

**Comment by michael-buschbeck** [comment]
Okay.

I'll still do something that makes the result of stringifying of `chat` less useless – e.g. make it stringify to `"chat(p1,p2)"`, or in case of custom functions, the same but with the actual parameter names.
